tests/test_api_build.py

```diff
@@ -1588,8 +1513,7 @@ def test_warning_on_file_clobbering(testing_config, capfd):
             api.build(os.path.join(recipe_dir, 'b'), config=testing_config)
 
 
-@pytest.mark.sanity
-@pytest.mark.serial
```

Comment:
I'm assuming this test passes when run? If so, leave it.

conda-verify is effectively deprecated because it isn't actively supported but this has never been officially announced. We ultimately do not know anything about how widely it may still be used so for now we should still test it.

We should instead open a new issue to investigate conda-verify's future.